projects/uitdatabank/models/common-offer-contactPoint.json

```diff
@@ -0,0 +1,54 @@
+{
+  "title": "contactPoint",
+  "type": "object",
+  "description": "Contact info containing one or more phone numbers, email addresses and/or website URLs.",
+  "examples": [
+    {
+      "phone": [
+        "+32/1234567890"
+      ],
+      "email": [
+        "info@example.com"
+      ],
+      "url": [
+        "https://www.example.com"
+      ]
+    }
+  ],
+  "properties": {
+    "phone": {
+      "type": "array",
+      "description": "List of phone numbers for contact purposes (can be empty)",
+      "minItems": 0,
+      "items": {
+        "type": "string",
+        "description": "Phone number for contact purposes. Can be an empty string, if no phone numbers are avaialble"
```

Comment:
Maybe it's important to mention that an empty string will clear existing values.